docs/source/app_developers_guide/ubuntu.rst

```diff
@@ -2,488 +2,716 @@
 
-One of the on-chain settings is the list of supported transaction families.
-To configure this setting, use the following steps to
-create and submit the batch to change the transaction family settings.
+   The ``intkey-tp-python`` transaction processor continues to run and to
+   display log messages in its terminal window.
 
-In this example, the ``sawset`` command specifies a JSON array
-which creates and submits a batch of transactions containing the
-settings change. This JSON array tells the validator or validator network
-to accept transactions of the following types:
+#. Start the XO transaction processor, ``xo-tp-python``. This transaction
+   processor will be used in a later tutorial.
 
-* ``intkey`` (IntegerKey transaction family)
-* ``sawtooth_settings`` (Settings transaction family)
+   #. Open a new terminal (the xo terminal window). The prompt ``user@xo-tp$``
+      shows the commands that should be run in this window.
 
-To create and submit the batch containing the new settings,
-enter the following command:
+   #. Run the following command:
 
-.. code-block:: console
+      .. code-block:: console
 
-  $ sawset proposal create sawtooth.validator.transaction_families='[{"family": "intkey", "version": "1.0"}, {"family":"sawtooth_settings", "version":"1.0"}]'
+         user@xo-tp$ sudo -u sawtooth xo-tp-python -v
 
-The output in the validator terminal window includes TP_PROCESS_REQUEST
-messages and information on the authorized keys and transaction families,
-as in this truncated example:
+   #. Check the validator terminal window to confirm that the transaction
+      processor has registered with the validator.
 
-.. code-block:: console
+      .. code-block:: console
 
-  ...
-  [2017-12-05 22:42:46.269 DEBUG    tp_state_handlers] GET: [('000000a87c...\n&sawtooth.settings.vote.authorized_keys\x12B0251fd...
-  ...
-  [2017-12-05 22:42:46.274 DEBUG    tp_state_handlers] GET: [('000000a87c...'sa    wtooth.validator.transaction_families\x12Y[{"family": "intkey", "version": "    1.0"}, {"family":"sawtooth_settings", "version":"1.0"}]')]
-  ...
-  [2017-12-05 22:52:33.495 DEBUG    interconnect] ServerThread sending TP_PROCESS_REQUEST to b'1893abb39b4b4aae'
-  ...
+         [2018-03-14 16:04:18.706 INFO     processor_handlers] registered transaction processor: connection_id=c885e99a11724e04e7da4ee426ee00d4af2cb54b67bf2fbd2f57e862bf28fa2c759a0d0978573782369659124797cc6f38d41bfde2469fe69e7e48dc1fadf5a9, family=xo, version=1.0, namespaces=['5b7349']
 
+   The ``xo-tp-python`` transaction processor continues to run and to display
+   log messages in its terminal window.
 
-Creating and Submitting Transactions
-====================================
+Other Start-up Options for the Transaction Processors
```

Comment:
To be consistent with the validator option change above, I'll delete this section.  I'll move the starting-order note to the top of the TP section. Note that the --bind info will move to the new section on adding nodes to an existing validator network.